src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
```

Comment:
The 'user' prop is not validated. It's highly recommended to validate your props for all incoming props for a component. You can use the 'prop-types' library to define the type of props the component expects, which can help catch bugs and prevent problems.